lib/github/middleware.js

```diff
@@ -35,6 +35,11 @@ const isFromIgnoredRepo = (payload) =>
   // Repository: https://admin.github.com/stafftools/repositories/seequent/lf_github_testing
   payload.installation.id === 491520 && payload.repository.id === 205972230;
 
+const isStateChangeAction = (action) => [
+  'opened',
```

Comment:
@atrigueiro in the Jira issue you mention that we should add `edited` to this list. Is that no longer true? Why can these three events trigger bot loops, but not any others?

Reply:
Yeah I've added an explanation for it on the description of this PR.

- The act of merging - which is inherently a one-time action - won’t cause a loop so the bot-filter can be dropped for merge hooks, but left in place for other types of hooks so that the status could be properly posted in JIRA.

- The infinite loop issue may only happen on comment/description edition because it is the only case where this Jira integration actually writes data in GitHub and can trigger events. For that reason, we won’t add edited to the list of actions. I.e. If two bots are both set to auto-respond to all comments, you'd get an infinite loop.